Sem8_Homework/UI.py

```diff
@@ -0,0 +1,74 @@
+import sys
+import view
+
+
+def add_row() -> list:
+    res = []
+    titles = ['ФАМИЛИЮ', 'ИМЯ', 'ДОЛЖНОСТЬ', 'ГОД РОЖДЕНИЯ', 'РАБ. ТЕЛЕФОН', 'ДОМ. ТЕЛЕФОН', 'МОБ. ТЕЛЕФОН']
+    for pos in titles:
+        val = input(f'Введите {pos}:   ').strip().title()
+        val = val if val else 'нет данных'
+        res.append(val)
+
+    return res
+
+
+def get_value(data: str) -> str:
+    return input(f'Введите {data}: ').strip().title()
+
+
+def menu() -> str:
+    while True:
+        view.row_view("""\n
+        [MENU]  Выберите действие:
+        <1> - посмотреть все записи
+        <2> - найти запись по фамилии
+        <3> - найти запись по должности
+        <4> -  добавить запись
+        <5> -  редактировать запись
+        <6> -  удалить запись
+        ------------------------------------
+        <q> или <Enter> - выйти из программы
+        """)

```

Comment:
Отступы великоваты, слева и сверху. Выглядит не очень красиво.
![image](https://user-images.githubusercontent.com/49741194/206111334-9737b1a2-2d11-4d21-ab65-de8c3754e6f5.png)
```suggestion
    while True:
        view.row_view('[MENU]  Выберите действие:\n'
                      '<1> - посмотреть все записи\n'
                      '<2> - найти запись по фамилии\n'
                      '<3> - найти запись по должности\n'
                      '<4> -  добавить запись\n'
                      '<5> -  редактировать запись\n'
                      '<6> -  удалить запись\n'
                      '- -----------------------------------\n'
                      '<q> или <Enter> - выйти из программы\n'
                      )
```
Так будет получше.

Reply:
Более того, все это сообщения. Можно поместить в 34 строке, внутри функции `input`.